backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/database/Schema.kt

```diff
@@ -13,7 +13,7 @@ const val CARD_INFO_HASH_LENGTH = 32 // Using SHA256-HMAC
 const val TOTP_SECRET_LENGTH = 20
 
 object Cards : IntIdTable() {
-    val totpSecret = binary("totpSecret", TOTP_SECRET_LENGTH)
+    val totpSecret = binary("totpSecret", TOTP_SECRET_LENGTH).nullable()
```

Comment:
See discussion in activation.ts

Reply:
We could also have another column `codeType` with values `STATIC | DYNAMIC` and a DB constraint that checks `codeType == STATIC <=> TOTPSECRET == NULL`
This would make it more explicit when we are dealining with a static vs a dynamic card.